### PR TITLE
Add printing of the actual send window.

### PR DESCRIPTION
**Pull request by he32**
To fit within 80 coloumns, shorten the printed units.
This is a suggested fix for issue #249.

### CODE REVIEW

File: src/units.c [hunk 1]
```diff
@@ -330,6 +331,26 @@ extern    "C"
...
+    void short_unit_snprintf(char *s, int inLen,
+                               double inNum, char inFormat)
+    {
+       char *cp;
+
+       unit_snprintf(s, inLen, inNum, inFormat);
+       if ((cp = index(s, 'i')) != NULL) {
+           *cp = 0;
+       }
+       if ((cp = index(s, 'y')) != NULL) {
```
Comment (pprindeville) [inline src/units.c]: Should this be an `else if` instead?

**Comment by pprindeville** [comment]
Is this PR being maintained?  Can it be rebased?

File: src/units.c (hunk 1 above)
Comment (he32) [inline src/units.c]: Could be, as a minor optimization.

**Comment by he32** [comment]
I just updated/merged what is in my branch.
I just got the impression there was little or no interest since the "send window"
information isn't available on Linux(?)

**Comment by bmah888** [comment]
I am going to, somewhat reluctantly, close this pull request as-is.  Thanks for working on this and submitting it.

There are three reasons for not taking it:

1.  Usefulness of the semantics of `tcp_info.tcpi_snd_wnd`.  I think that when we looked into this, it was the window advertisement that was being advertised to the other side, so the number on the client side only makes sense in the --reverse case where the server is sending.  At this point I don't think we know how to get the number originally requested in #249.

2.  We'd prefer not to make any changes to the human-readable output at this point, and only have the window size output, if we can get it, in the JSON.

3.  Not working on Linux (or mac OS), thus limiting its applicability.  Of our three main development platforms, this only worked on FreeBSD.

**Pull request closed without merging**